_pages/en_US/seedminer-(alternate).txt

```diff
@@ -0,0 +1,34 @@
+---
+title: "Seedminer (Alternate)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% include_relative include/seedminer-base.txt %}
+
+___
+
+### Next steps
+
+Select the method based on the application that you have installed on your 3DS.
```

Comment:
May want to add a note here that no method here is necessarily "better" than the other and will have the same result in the end (granted, DSiWare is probably ultimately easier than pichaxx/steelhax)